### PR TITLE
Change default rst indentation to 4 spaces

### PR DESCRIPTION
**Pull request by linawolf**
refs https://github.com/TYPO3-Documentation/T3DocTeam/issues/194

### CODE REVIEW

**Comment by oliverklee** [comment]
Thank you, @linawolf! I think this change deserves a CHANGELOG entry (here in the tea project). Could you please add one? Thanks!

**Comment by linawolf** [comment]
In the sense of the Core this would be a TASK and deserve no Changelog entry. How come you would want it in the changelog?

**Comment by oliverklee** [comment]
In the tea project (which serves as an example and as a template/copy’n’paste quarry), we usually put those kinds of things in the changelog:
- new features
- bug fixes
- changes of notice that would also require/encourage changes in the projects that use this extension as an example

So the change in the `.editorconfig` falls in the last category.

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Changed
 - Move npm tools and config to default locations (#444)
 - Use the TYPO3 Code of Conduct (#430)
+- Change the default indentation for rst files to 4 spaces (https://github.com/TYPO3-Documentation/T3DocTeam/issues/194)
```
Comment (oliverklee) [inline CHANGELOG.md:28]: Now that 2.0.0 is out, this needs to go into the changelog for the upcoming version (x.y.z), not 2.0.0 anymore.

Comment (linawolf) [reply CHANGELOG.md]: done

**Review by oliverklee — APPROVED** [review]
Thanks, @linawolf!

**Pull request merged**